src/plugins/intel_cpu/src/node.cpp

```diff
@@ -823,7 +823,7 @@ void Node::updateDynamicParams() {
 }
 
 void Node::execute(const dnnl::stream& strm, int numaId) {
-    OV_CPU_NODE_SCOPED_TASK_BASE(getTypeStr());
+    OV_ITT_SCOPED_TASK_BASE(itt::domains::ov_intel_cpu, profiling.execute);
```

Comment:
Shouldn't we initialize profiling.execute with a type instead of a name?
I guess we can also just bring back original logic of the macro VERBOSE_PERF_DUMP_ITT_DEBUG_LOG, which did use profiling.execute.
And just drop this one completely